src/components/token_invalid/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Trans, withTranslation } from 'react-i18next';
+
+import './index.scss';
+import IrmaAppBar from '../../widgets/irma_app_bar';
+import Column from '../../widgets/column';
+import IrmaButton from '../../widgets/irma_button';
+
+const TokenInvalid = (props) => {
+  const onRetry = () => {
+    props.dispatch({ type: 'resolveError' });
+  };
+
+  return (
+    <>
+      <IrmaAppBar title={props.t('header')} />
+      <Column>
+        <p>{props.t('explanation')}</p>
+        <p>{props.t('explanation-details')}</p>
+        <ul>
+          <li>
+            <Trans
+              t={props.t}
+              i18nKey="point-1"
+              // eslint-disable-next-line
+              components={[<a href={undefined}
```

Comment:
What does this do exactly?

Reply:
To start with, we need a javascript handler when clicking on the link. If you use:

`<a href="javascript:...">` it would produce the following warning:
`index.js:1 Warning: A future version of React will block javascript: URLs as a security precaution. Use event handlers instead if you can. If you need to generate unsafe HTML try using dangerouslySetInnerHTML instead. React was passed "javascript:;" `

If you would do something like `<a href="#" onClick={...}>` it would invoke an unwanted 'url change', causing the onClick event not to be fired.

About the `// eslint-disable-next-line`. This is done for readability, otherwise we would have to format it in such a way it is correct, but thereby becomes difficult to read.